markdown/pxf/ReadWritePXF.html.md.erb

```diff
@@ -34,7 +34,7 @@ PXF comes with a number of built-in profiles that group together a collection o
 -   HBase (Read only)
 -   JSON (Read only)
 
-You can specify a built-in profile when you want to read data that exists inside HDFS files, Hive tables, HBase tables, and JSON files and for writing data into HDFS files.
+You can specify a built-in profile when you want to read data that exists inside HDFS files, Hive tables, HBase tables, and JSON files and when you want to write data into HDFS files.
```

Comment:
Add a comma after "JSON files,"   Also not sure but it seems like the "and" should change to "or" in this section.